app/views/admin/users/show.html.haml

```diff
@@ -1,18 +1,44 @@
+= render 'admin/nav', {active: 'users'}
+
 %h1
   = @user.name
   - if current_user.has_role?(:admin)
     &nbsp;
     %small
       = link_to 'Edit', edit_admin_user_path(current_user), class: 'btn btn-default'
 
+- if @user.description.present?
+  %p
+    %b
+      Description:
+    %div
+      = @user.description
+
 %p
-  @user.description
+  %b
+    Admin?: <b>#{@user.has_role?(:admin)}</b>
 
 %p
-  Roles:
+  Dispatcher for Zones:
+  %ul
+    - if @dispatching_for.present?
+      - zones.each do |rz|
+        %li
+          = rz.name
+    - else
+      %li
+        %i
+          None
+
 
 %p
-  Driving for Ride Zones:
-  - RideZone.find_roles(:driver, @user).each do |rz|
-    %div
-      = rz.name
+  Driving for Zones:
+  %ul
+    - if @driving_for.present?
```

Comment:
We explicitly decided to not support multiple zones per driver, and there's logic in ride_zones_controller#add_role to enforce that. I think if someone really needs to do it we can ask them to create a separate account.